src/factors/Inertial/IMUDeltaFactor.jl

```diff
@@ -324,7 +310,7 @@ Base.@kwdef struct IMUDeltaFactor{T <: SamplableBelief} <: AbstractManifoldMinim
     J_b::SMatrix{10,6,Float64} = zeros(SMatrix{10,6,Float64})
     # accelerometer bias, gyroscope bias 
     b::SVector{6, Float64} = zeros(SVector{6, Float64})
-    #optional raw measurements
+    #optional raw measurements, FIXME replace with BlobEntry -- dont do abstract types here, or raw data if not needed in hotloop 
```

Comment:
What abstract types?

Reply:
does this not create an ablstract type raw_measurement::Union?

Reply:
Both Nothing and IMUMeasurement are concrete types. Union{Nothing, T} are normally not a problem